### ISSUE TITLE
[Feature Request] Ability to default the commit details view to other than "Commit"

### ISSUE THREAD

**Issue by bsarrazin** [ForkIssues/Tracker#573]
The default is "Commit"
![image](https://user-images.githubusercontent.com/1077441/54298351-f59eac00-458e-11e9-8f52-f2f61ca7485b.png)

It reverts back to "Commit" every so often (I didn't keep track of how/when it reverts back). It would be nice to have the ability to set the default to "Changes" or "File Tree" since I rarely use the "Commit" view.

Thank you for considering this small feature!

**Comment by DanPristupov** [comment]
The active tab should is never changed automatically, however the Commit is open by default.

Regarding the feature:

It doesn't make much sense to allow to select "File Tree" as a default tab.

May be it will be better to remember the active tab and restore it after restart?

**Comment by maksimovic** [comment]
> May be it will be better to remember the active tab and restore it after restart, what do you think?

Yes, please 🍻

**Comment by bsarrazin** [comment]
@DanPristupov that makes sense to me.
Yes, remember the active tab would be great! 🙏

**Comment by hgustafsson** [comment]
I agree, remembering the active tab between sessions/restarts would be great. I would love to be able to have the "Changes" tab open by default in this way. 

I wouldn't mind if this was remembered "globally" for all repositories rather than have to remember a per repository setting. Then new repositories would open with the same default tab.

**Comment by bsrz** [comment]
I don't expect this to make it into the app at this point. It has been 2+ years at this point. Closing.
